src/SelfService/Application/ConfigurationLevelService.cs

```diff
@@ -0,0 +1,116 @@
+using SelfService.Domain.Models;
+using System.Linq;
+
+namespace SelfService.Application;
+
+public enum ConfigurationLevel
+{
+    None,
+    Partial,
+    Complete
+}
+
+public class ConfigurationLevelDetail : Entity<ConfigurationLevelDetail>
+{
+    public ConfigurationLevel level { get; set; }
+    public string description { get; set; }
+    public string suggestion { get; set; }
+    public bool isFocusMetric { get; set; }
+
+    public ConfigurationLevelDetail(ConfigurationLevel level, string description, string suggestion, bool isFocusMetric)
+    {
+        this.level = level;
+        this.description = description;
+        this.suggestion = suggestion;
+        this.isFocusMetric = isFocusMetric;
+    }
+}
+
+public class ConfigurationLevelInfo : Entity<ConfigurationLevelInfo>
+{
+    public ConfigurationLevel overallLevel { get; set; }
+    public bool IsFocusMetric { get; set; }
```

Comment:
`IsFocusMetric` is probably not important for the overall information; Just for the Subservient LevelDetails

Reply:
agreed